docs/learn/concepts.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: '📑 Major Concepts'
+sidebar_position: 2
+---
+
+# Major Concepts
+
+## Universal Profile (UP):
```

Comment:
Is the colon ":" needed at the end?

As it appears like that in the sidebar. I would suggest removing it, but this is just my personal preference and I am not too strong about it.

<img width="1396" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/11788e36-77ac-4676-aa27-3e57008cf1bb">

Reply:
removed